// const myPromise = new Promise((resolve, reject ) => {
//     setTimeout(()=>{
//         const number = Math.random();
//         if (number > 0.5){
//             resolve(number);
//         }
//         else{
//             reject("FEILMELDING");
//         }
//     },1000);
// });
// myPromise
// .then((result) => {
//     console.log(result); 
// })
// .catch((error) => {
//     console.log(error);
// });

// try{
//     console.log(MinVariableSomIkkeEksisterer);
// } catch (err) {
//     console.log(err);
// } finally {
//     console.log("DETTE KJÃ˜RES UANSETT IN DET ER FEIL ELLER IKKE");
// }


